website/static/getting-started.html

```diff
@@ -392,12 +392,30 @@ <h3>Asynchronous actions</h3>
                         </p>
 
                         <h3>DevTools</h3>
+                        <p>
+                            Following tools can be analyzed to analyze
+                            your mobx-react application:
+                        </p>
+                        <ul>
+                            <li>
+                                Visualizing re-rendering of components is now part of the standard
+                                <a href="https://reactjs.org/blog/2019/08/15/new-react-devtools.html" target="_blank">React devtools</a>
+                            </li>
+                            <li>
+                                The dependency tree of a component tree can be inspected by showing the state of the <code>useObserver</code>
+                                hook in the React devtools (older versions may display it as just an <code>Object</code>)
+                            </li>
+                            <li>
+                                Spying on events can still be done with the
+                                <a href="https://github.com/mobxjs/mobx-devtools" target="_blank">MobX-react browser plugin</a>
+                                through the <a href="https://github.com/winterbe/mobx-logger" target="_blank">mobx-logger package</a>
+                                or manually by using the <code>spy</code> or <code>trace</code> utility from the mobx package.
+                            </li>
+                        </ul>
                         <p>
                             The <a href="https://github.com/mobxjs/mobx-react-devtools" target="_blank"><code>mobx-react-devtools</code></a> package
```

Comment:
Please remove that duplicate `package` word, it's already in following brackets.